codeboarding_cli/commands/remote_analysis.py

```
@@ -0,0 +1,84 @@
+
+            try:
+                process_remote_repository(
+                    repo_url=repo,
+                    run_id=run_context.run_id,
+                    log_path=run_context.log_path,
+                    output_dir=repo_output_dir,
+                    depth_level=args.depth_level,
+                    upload=args.upload,
+                    monitoring_enabled=should_monitor,
+                )
+            except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)